app/views/devise/registrations/edit.html.erb

```diff
@@ -46,7 +46,7 @@
 <% if current_user.role == "standard" %>
   <%= link_to "Upgrade to Premium Now", new_charge_path %>
 <% elsif current_user.role == "premium" %>
-  <%= link_to "Downgrade to standard", user.downgrade_role  %>
+  <%= link_to "Downgrade to standard", current_user.downgrade_role  %>
```

Comment:
This currently results in an error, when we click the link.  We should link to a __path__ instead (that should be defined in the Charges controller to downgrade the user)